docs/install/ubuntu_setup.md

```diff
@@ -1,67 +1,103 @@
 # Installing MXNet on Ubuntu
 
-**NOTE:** For MXNet with Python installation, please refer to the [new install guide](http://mxnet.io/install/index.html).
+The following installation instructions are for installing MXNet on computers running **Ubuntu 16.04**. Support for later versions of Ubuntu is [not yet available](#contributions).
+<hr>
 
-MXNet currently supports Python, R, Julia, Scala, and Perl. For users of R on Ubuntu operating systems, MXNet provides a set of Git Bash scripts that installs all of the required MXNet dependencies and the MXNet library.
+## Contents
 
-The simple installation scripts set up MXNet for Python and R on computers running Ubuntu 12 or later. The scripts install MXNet in your home folder ```~/mxnet```.
+* [CUDA Dependencies](#cuda-dependencies)
+* [Quick Installation](#quick-installation)
+    * [Python](#install-mxnet-for-python)
+    * [pip Packages](#pip-package-availability)
+* [Standard Installation](#standard-installation)
+* [Installing Language Packages](#installing-language-packages-for-mxnet)
+    * [R](#install-the-mxnet-package-for-r)
```

Comment:
Is the CPP package missing?

Reply:
You're right... this needs to be added to this page.